### DISCUSSION TITLE
Also working for non-public Variables?

### DISCUSSION THREAD

**Discussion by LeopoldLerch** [expatfile/next-runtime-env#75] [Q&A] [answered]
Hi
I understand, that it will fetch the "NEXT_Public" prefixed variables at runtime. But will it also do this for non-public-variables which are supposed to be used only on server side and just not meant to go to the client? 

In the examples server only variables are only used in "getstaticprops", which is only executed for generating static stuff during build time.

**Comment by HofmannZ** [comment, marked as answer]
You can use the [makeEnvPublic](https://github.com/expatfile/next-runtime-env/discussions/75#makeenvpublickey-string--string-void) utility for this.
